### DISCUSSION TITLE
Language support

### DISCUSSION THREAD

**Discussion by olivierlesnicki** [rainbow-me/rainbowkit#327] [Ideas]
Is there a plan to support other languages?

**Comment by nickbytes** [comment]
We've chatted briefly about this and are definitely interested. We'll be chatting about it over the next few weeks as we plan out our roadmap. We'd also love to see a proposal from the community on how they envision this might work!

**Comment by DanielSinclair** [comment]
Now available with [v1.1.0](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%401.1.0). This will work automatically based on a user's preferred browser language, but you can reference our docs for best practices and implementation details for more advanced approaches: https://www.rainbowkit.com/docs/localization
